src/sentences/txt.rs

```diff
@@ -0,0 +1,62 @@
+use nom::character::complete::char;
+
+use nom::{bytes::complete::take_while, IResult};
+
+use super::utils::number;
+use crate::parse::NmeaSentence;
+
+/// Parse TXT message from u-blox device
+///
+/// $GNTXT,01,01,02,u-blox AG - www.u-blox.com*4E
```

Comment:
It would be nice to have basic test, may be with help of this example message?
So this functionality doesn't become broken after some future refactoring

Reply:
I added a test, thanks!